apps/docs/pages/docs/api-reference/fields/richtext.mdx

```diff
@@ -0,0 +1,733 @@
+| [`menu`](#menu)                           | `menu: { typography: ["Bold", "Italic", "Underline", "Strikethrough"] }`            | `Record<string,string[]>`                                      | -        |
+| [`inlineMenu`](#inlineMenu)               | `inlineMenu: { typography: ["Bold", "Italic", "Underline", "Strikethrough"] }`      | `Record<string,string[]>`                                      | -        |
+| [`controls`](#controls)                   | `controls: { MyControl: {render: (editor, editorState) => <div>My Control</div>} }` | `Record<string, {render: (editor, editorState) => ReactNode}>` | -        |
+| [`textSelectOptions`](#textselectoptions) | `textSelectOptions: ["p","h2"]`                                                     | `("p" \| "h2" \| "h3" \| "h4" \| "h5" \| "h6")[]`              | -        |
+| [`extensions`](#extensions)               | `extensions: [StarterKit, Link]`                                                    | `Extensions[]` <i>from TipTap</i>                              | -        |
+| [`selector`](#selector)                   | `selector: (ctx) => ({ ... })`                                                      | `RichTextSelector`                                             | -        |
+
+## Required params
+
+### `type`
+
+The type of the field. Must be `"richtext"` for RichText fields.
+
+```tsx {6} copy
+const config = {
+  components: {
+    Example: {
+      fields: {
+        description: {
+          type: "richtext",
+        },
+      },
+      // ...
+    },
+  },
+};
+```
+
+## Optional params
+
+### `contentEditable`
+
+Enable inline text editing for this field.
+
+```tsx {7} copy
+const config = {
+  components: {
+    Example: {
+      fields: {
+        description: {
+          type: "richtext",
+          contentEditable: false,
+        },
+      },
+      // ...
+    },
+  },
+};
+```
+
+### `menu`
+
+Provide an override for the grouped button layout rendered in the editor toolbar. Each key in your `menu` matches a menu group and maps to an array of control keys.
+
+```tsx {7-10} copy
+const config = {
+  components: {
+    Example: {
+      fields: {
+        description: {
+          type: "richtext",
+          menu: {
+            typography: ["Bold", "Italic", "Underline"],
+            blocks: ["BulletList", "OrderedList"],
+          },
+        },
+      },
+    },
+  },
+};
+```
+
+### `inlineMenu`
+
+Provide an override for the grouped button layout rendered in the `BubbbleMenu` when inline editing of your field. Each key in your `inlineMenu` matches a menu group and maps to an array of control keys.
+
+```tsx {7-10} copy
+const config = {
+  components: {
+    Example: {
+      fields: {
+        description: {
+          type: "richtext",
+          inlineMenu: {
+            typography: ["Bold", "Italic", "Underline"],
+            blocks: ["BulletList", "OrderedList"],
+          },
+        },
+      },
+    },
+  },
+};
+```
+
+### `controls`
```

Comment:
Suggest we need a list of "Default controls" underneath this, rather than under "Defaults"